index.html

```diff
@@ -160,10 +160,10 @@ <h1>Frequently asked questions</h1>
 				<div class="col-md-5">
 					<div class="faq-question">
 								<h2 class="text-colored">What's so special about HackHPI?</h2>
-								<p>There are very few health tech hackathons striving to build useful applications with raw data, and we want to change that! For you it will be a chance to get in contact with our awesome partners and experiment with their data and gadgets.</p>
+								<p>There are very few hackathons about environment striving to build useful applications with raw data, and we want to change that! For you it will be a chance to get in contact with our awesome partners and experiment with their data and gadgets.</p>
```

Comment:
`about environment` sounds a little odd to me - what do you think @jak-ing?

Reply:
Yes. I think we need to change the whole sentence here since we are not focussing on building stuff with 'raw data' either this year.